notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaEntry.java

```diff
@@ -229,17 +223,22 @@ public void addAction(JpaAction attribute) {
      */
     public void setAddressees(Set<JpaAddressee> addressees) {
         this.addressees.clear();
-        for (JpaAddressee addressee : addressees) {
-            addAddressee(addressee);
-        }
+        this.addressees.addAll(addressees);
     }
 
     /**
      * Adds the specified addressee to the current collection.
      */
     public void addAddressee(JpaAddressee addressee) {
         addressees.add(addressee);
-        addressee.setEntryId(getId());
+    }
+
+    @Override
+    public String toString() {
+        return "JpaEntry [id=" + id + ", title=" + title + ", source=" + source + ", category=" + category + ", url="
```

Comment:
would [`org.apache.commons.lang3.builder.ToStringBuilder`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html) be a more effective way to generate this string?